.github/deploy.sh

```diff
@@ -34,6 +34,8 @@ git config user.name "GHA CI"
 git config user.email "gha@ci.invalid"
 
 if [[ -n $TAG_NAME ]]; then
+  # track files, so that the following check works
+  git add --intent-to-add "$TAG_NAME"
```

Comment:
til about `--intent-to-add` :exploding_head:

Reply:
Same 😄 the short variant is `-N`